apps/server/src/modules/tldraw/controller/tldraw.ws.ts

```diff
@@ -45,4 +65,13 @@ export class TldrawWs implements OnGatewayInit, OnGatewayConnection {
 		const urlStripped = request.url.replace(/(\/)|(tldraw-server)/g, '');
 		return urlStripped;
 	}
+
+	public parseCookiesFromHeader(request: Request): { [p: string]: string } {
+		const parsedCookies: { [p: string]: string } = cookie.parse(request.headers.cookie || '');
+		return parsedCookies;
+	}
+
+	public closeClient(client: WebSocket, code: WsCloseCodeEnum, data: string): void {
```

Comment:
same here, should be private method

Reply:
this method is public for testing purposes